Runtimes/Supplemental/Synchronization/CMakeLists.txt

```diff
@@ -0,0 +1,106 @@
+cmake_minimum_required(VERSION 3.29)
+
+if($ENV{BUILD_NUMBER})
+  math(EXPR BUILD_NUMBER "$ENV{BUILD_NUMBER} % 65535")
+  set(BUILD_NUMBER ".${BUILD_NUMBER}")
+endif()
+
+project(SwiftSynchronization
+  LANGUAGES Swift
+  VERSION 6.1.0${BUILD_NUMBER})
+
+if(NOT PROJECT_IS_TOP_LEVEL)
+  message(SEND_ERROR "Swift Synchronization must build as a standalone project")
+endif()
+
+set(CMAKE_POSITION_INDEPENDENT_CODE YES)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/../cmake/modules")
+
+set(CMAKE_Swift_LANGUAGE_VERSION 5)
+
+include(GNUInstallDirs)
+
+set(${PROJECT_NAME}_SWIFTC_SOURCE_DIR
+  "${PROJECT_SOURCE_DIR}/../../../"
+  CACHE FILEPATH "Path to the root source directory of the Swift compiler")
+
+set(${PROJECT_NAME}_VENDOR_MODULE_DIR "${CMAKE_SOURCE_DIR}/../cmake/modules/vendor"
+  CACHE FILEPATH "Location for private build system extension")
+
+find_package(SwiftCore QUIET REQUIRED)
+
+include(gyb)
+include(ResourceEmbedding)
+include(AvailabilityMacros)
+include(PlatformInfo)
+
+include("${${PROJECT_NAME}_VENDOR_MODULE_DIR}/Settings.cmake" OPTIONAL)
+
+option(${PROJECT_NAME}_ENABLE_LIBRARY_EVOLUTION "Generate ABI resilient runtime libraries"
+  ${SwiftCore_ENABLE_LIBRARY_EVOLUTION})
+
+option(${PROJECT_NAME}_ENABLE_PRESPECIALIZATION "Enable generic metadata prespecialization"
+  ${SwiftCore_ENABLE_PRESPECIALIZATION})
+
+include(CatalystSupport)
+include(EmitSwiftInterface)
```

Comment:
We likely need to import `InstallSwiftInterface` to be able to call `install_swift_interface`

```
include(InstallSwiftInterface)
```